### DISCUSSION TITLE
HELP: qiankun+Angular the font  images in the css of micro-app load 404

### DISCUSSION THREAD

**Discussion by Ivy1994** [umijs/qiankun#1592] [Q&A] [answered]
I've updated the webpack config.js according to the soluation given by the qiankun webiste. However, the image in the micro-app still load 404...So, I was wondering if anyone can help me to fix this issue.Thank you~
PS： The Angular version is 7.0.0 and the configuration is as follows
```js
 rules: [
      {
        test: /\.(png|jpe?g|gif|svg)(\?.*)?$/,
        use: [
           {
             loader: 'url-loader',
             options: {
               limit: 1000000
             }
           }
         ]
      }
    ]
```

**Comment by kuitos** [comment, marked as answer]
check the [faq doc](https://qiankun.umijs.org/faq#why-dynamic-imported-assets-missing)
